tsconfig.spec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "types": ["jest", "node"]
+  },
+  "include": [] // test speed boost trick
```

Comment:
Do you want to commit this test to master?

Reply:
oh this one means “boost test speed” 😅

Reply:
updated comment to be less confused